backends/advanced/src/advanced_omi_backend/main.py

```
@@ -50,6 +50,10 @@
 )
 from advanced_omi_backend.audio_utils import process_audio_chunk
 from advanced_omi_backend.task_manager import init_task_manager, get_task_manager
+
+from advanced_omi_backend.rq_queue import redis_conn
+
+from advanced_omi_backend.task_manager import init_task_manager
```

Ruff (0.13.1)

Redefinition of unused `init_task_manager` from line 52

Remove definition: `init_task_manager`

(F811)